.github/workflows/upload-perf-results.yml

```diff
@@ -93,3 +93,19 @@ jobs:
         export PGPORT=$(cat ${{github.workspace}}/PGPORT)
         export PGDATABASE=$(cat ${{github.workspace}}/PGDATABASE)
         psql -f ${{github.workspace}}/results/upload.sql
+
+    - name: Download and run regression script
+      run: |
+        curl https://raw.githubusercontent.com/microsoft/bpf_performance/refs/heads/main/scripts/check_perf_results.sql > check_perf_results.sql
+        export PGPASSWORD=$(cat ${{github.workspace}}/PGPASSWORD)
+        export PGHOST=$(cat ${{github.workspace}}/PGHOST)
+        export PGUSER=$(cat ${{github.workspace}}/PGUSER)
+        export PGPORT=$(cat ${{github.workspace}}/PGPORT)
+        export PGDATABASE=$(cat ${{github.workspace}}/PGDATABASE)
+        psql -f check_perf_results.sql --csv > ${{github.workspace}}/results/regression_results.csv
```

Comment:
if we are providing parameters to check_perf_results.sql as part of this pull request ('platform', 'max_sigma') --- https://github.com/microsoft/ebpf-for-windows/pull/4029 , shouldn't we pass in those parameters here?

Reply:
Will do once I have updated the script